plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -2893,6 +2895,26 @@ public void testUnregisterPartitionWithNullArgument()
         assertQueryFails("CALL system.unregister_partition('web', 'page_views', ARRAY['country'], NULL)", ".*partition_values cannot be null.*");
     }
 
+    @Test
+    public void testUnregisterTable()
```

Comment:
Can you add a test for unregistering an external table as well?
Also a partitioned table

Also a test for disallowing unregistering a view would be welcomed.